inc/woocommerce/class-storefront-woocommerce.php

```diff
@@ -114,13 +114,16 @@ public function woocommerce_scripts() {
 		 * @since 1.6.0
 		 */
 		public function star_rating_script() {
-			if ( wp_script_is( 'jquery', 'done' ) && is_product() ) {
+			if ( is_product() ) {
 		?>
 			<script type="text/javascript">
-				jQuery( function( $ ) {
-					$( 'body' ).on( 'click', '#respond p.stars a', function() {
-						var $container = $( this ).closest( '.stars' );
-						$container.addClass( 'selected' );
+				requestAnimationFrame(function() {
```

Comment:
Wasn't exactly sure where this script will be injected, so I've implemented it by assuming `p.stars` are already rendered by the next frame time (aka, no other stars elements are created dynamically at some later point)